adal/devApps/XFormsApp.Droid/SecondPageRenderer.cs

```diff
@@ -40,6 +40,8 @@ class SecondPageRenderer : PageRenderer
     {
         public SecondPageRenderer(Context context) : base(context)
         {
+            DependencyService.Register<DroidPlatformParametersFactory>();
```

Comment:
>DependencyService.Register<DroidPlatformParametersFactory>(); [](start = 12, length = 61)

could have been done without the dependency service. could have injected the IPlatformParameter instead thus not having the need for a static in the factory (or a factory at all for that matter). Statics are ok for this

Reply:
We can discuss this...open to suggestions. #MakeItGreat

Reply:
It's not good imo if it requires statics and DI can be done with-out a service, some times the simpler way is better (especially if it avoids static state). The static can be removed yes by creating several platform parameters... which this code also does though only the type is passed in. An alternative is a call back/delegate which sets the value. I will be more than happy to discuss in more details outside of the PR.